deps/node/src/node_main_lw_runner-inl.h

```diff
@@ -127,6 +127,10 @@ class LWNodeMainRunner {
 
     Context::Scope context_scope(env_->context());
 
+    if (on_context_created_callback_) {
```

Comment:
on_context_created_callback_

Since it's not a callback that is called when a context is created, it would be better to name it something that means it's called after the `MainEnvironment` is created.

Reply:
Okay. I will change it to `on_main_env_creation_callback`.